Frameworks/CoreLocation/CLGeocoder.mm

```diff
@@ -14,7 +14,7 @@
 //
 //******************************************************************************
 
-#import "Windows.h"
+#import "windows.h"
```

Comment:
Given it's all over the place, for (W/w)indows. it would be a change on it's own.